ozaria/site/styles/play/level/level-playback-view.sass

```diff
@@ -3,18 +3,16 @@
 @import "ozaria/site/styles/play/variables"
 
 #playback-view
-  $playback-button-color: rgb(248, 197, 146)
+  $playback-button-color: #ffffff
```

Comment:
This is a variable declaration in sass. Thanks @shubhi1092 for telling me this offline.